_episodes/08-collab.md

```diff
@@ -36,7 +36,11 @@ needs to go to [https://github.com/notifications](https://github.com/notificatio
 Once there she can accept access to the Owner's repo.
 
 Next, the Collaborator needs to download a copy of the Owner's repository to her
- machine. This is called "cloning a repo". To clone the Owner's repo into
+ machine. This is called "cloning a repo". 
+Important: do not place this copy inside your own planets folder!
```

Comment:
Maybe move this note below line 43's clone, and modify it to say something like "if you choose to clone without the clone path (`~/Desktop/vlad-planets`) specified, you will clone inside your own planets folder! Make sure to navigate to the `Desktop` folder first.

Reply:
I'm fine moving the comment. But I don't fully understand your suggestion "if you choose to clone without the clone path (~/Desktop/vlad-planets) specified". Suggestions in a new commit.